external/opal-prd/hostboot-interface.h

```diff
@@ -71,6 +84,61 @@ enum MemoryError_t
 #define HBRT_PM_LOAD    0
 #define HBRT_PM_RELOAD  1
 
+/** Common return codes for scom_read(), scom_write(). */
+#define HBRT_RC_RANGE__scom   0x1000
+
+/* RC for a piberr is equal to 0x1000 plus the pib error value,
+ made into a negative */
+#define HBRT_RC_PIBERR_MASK            (0x00000000u - 0x00001007u) /* 0xFFFF_EFF9 */
+
+#define HBRT_RC_PIBERR_001_BUSY        (0x00000000u - 0x00001001u) /* 0xFFFF_EFFF */
+#define HBRT_RC_PIBERR_010_OFFLINE     (0x00000000u - 0x00001002u) /* 0xFFFF_EFFE */
+#define HBRT_RC_PIBERR_011_PGOOD       (0x00000000u - 0x00001003u) /* 0xFFFF_EFFD */
+#define HBRT_RC_PIBERR_100_INVALIDADDR (0x00000000u - 0x00001004u) /* 0xFFFF_EFFC */
+#define HBRT_RC_PIBERR_101_CLOCKERR    (0x00000000u - 0x00001005u) /* 0xFFFF_EFFB */
+#define HBRT_RC_PIBERR_110_PARITYERR   (0x00000000u - 0x00001006u) /* 0xFFFF_EFFA */
+#define HBRT_RC_PIBERR_111_TIMEOUT     (0x00000000u - 0x00001007u) /* 0xFFFF_EFF9 */
+
```

Comment:
Again its nice if we get rid of numbers within macro.

Reply:
See answer above, the 01-07 have real meaning in the hardware since they are the actual piberr values from the chip.

Reply:
I got that. But why not something like 
  s/HBRT_RC_PIBERR_101_CLOCKERR/HBRT_RC_PIBERR_CLOCKERR/

Numbers within macro name makes it hard.

Reply:
I find the numbers extremely useful because that is the canonical representation of these PIB codes, especially since sometimes the same code is reused for a few different causes.